test/pg-cdc-resumption/mzcompose.yml

```diff
@@ -191,15 +213,15 @@ services:
     mzbuild: materialized
     command: --experimental --disable-telemetry
     ports:
-      - 6875
+      - 6875:6875
```

Comment:
I think host port selection should be left dynamic. Otherwise people that have materialize or postgres running locally will see conflicts.